content/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists.md

```diff
@@ -65,6 +65,12 @@ You can reorder tasks across different lists in the same comment, but you cannot
 
 {% ifversion fpt %}
 
+## Converting tasks into issues
+
+You can turn tasks into issues. First, hover to one of the checkboxes in your issue until a circle with a dot in the middle appears. Then, click on Convert to Issue.
+
+ {% ifversion fpt or ghec %} ![Screenshot of a {% data variables.product.prodname_dotcom %} issue showing two tasks. The first task has the issue symbol in green with its title "Convert text into issues" outlined in light blue.](/assets/images/help/writing/convert-task-lists-into-issues.png){% endif %}
```

Comment:
Just making a few small tweaks here, updating the screenshot alt text, and using an octicon so we don't need to visually describe the icon. I'll apply this suggestion myself 😸 

```suggestion
You can also convert tasks into issues. First, hover over one of the items in your task list and then click {% octicon "issue-opened" aria-label="Convert to issue" %}.

 {% ifversion fpt or ghec %} ![Screenshot of a {% data variables.product.prodname_dotcom %} issue showing two tasks. The"Convert to issue" icon is highlighted with an orange outline.](/assets/images/help/writing/convert-task-lists-into-issues.png){% endif %}
```